distribution/windows/install.nsi

```diff
@@ -201,23 +223,26 @@ Section "Uninstall"
     Delete "$SMPROGRAMS\$SHORTCUTS\Contributors.lnk"
 
     ; Clean up OpenRCT2 dir
+    Delete "$INSTDIR\curl-ca-bundle.crt"
     Delete "$INSTDIR\changelog.txt"
+    Delete "$INSTDIR\known_issues.txt"
+    Delete "$INSTDIR\licence.txt"
     Delete "$INSTDIR\readme.txt"
     Delete "$INSTDIR\contributors.md"
-    Delete "$INSTDIR\openrct2.exe"
-    Delete "$INSTDIR\openrct2.dll"
-    Delete "$INSTDIR\licence.txt"
+    Delete "$INSTDIR\${OPENRCT2_EXE}"
+!ifdef OPENRCT2_DLL
```

Comment:
What if user uses this installer to replace existing 32 or 64 bit installation with, respectively, 64 or 32 bit one?